### DISCUSSION TITLE
When run the GitHub Action, there is bug

### DISCUSSION THREAD

**Discussion by no13bus** [DevSkillsHQ/discusssions#23] [Help] [answered]
When the action go to the step:
**Publish test results. Attempt 1.
Publish test results. Attempt 2.**

It goes wrong.

<img width="1113" alt="Screen Shot 2023-05-07 at 20 36 42" src="https://user-images.githubusercontent.com/3690806/236696423-0ac620fd-32a9-44d1-ab81-3774817fab23.png">

It said that I do not have the auth Error: 
**Received status code: 401**

**Comment by no13bus** [comment]
This bug make my project can not pass the GitHub action. I do not know the reasons. Maybe I do not have the access to the webhook?

**Reply by no13bus** [reply]
@fenske Thanks. I saw you help me to fix it.
By the way, I suggested that you fix the bug for all the project template files.

**Reply by fenske** [reply, marked as answer]
Hi @no13bus. I'm Anton from Alva. Really sorry about this issue! We've been migrating away from the old platform called DevSkills into Alva Coding Tests and this was one of the things that didn't get properly migrated. Fixed it now for everyone affected thanks to your insightful comment! 🙏

I wish you the best of luck with the rest of your interview process!

BR,
Anton.

**Reply by no13bus** [reply]
This bug indeed delayed me some time to solve, and I hope it won't have too much impact on the interview result. I also wish you all the best.
